app/util/mapIconUtils.js

```diff
@@ -679,7 +671,7 @@ export function drawCitybikeIcon(
       tile.ctx.drawImage(image, x, y);
       x = x + width - smallCircleRadius;
       y += smallCircleRadius;
-      if (showAvailabilityBadge) {
+      if (showAvailability && bikesAvailable >= 0) {
         /* eslint-disable no-param-reassign */
```

Comment:
Why remove showAvailabilityBadge?

Reply:
It was cleaner, but now, with a more complex condition I'll put it back.